index.js

```diff
@@ -2,29 +2,79 @@
  * Q0: Write a function called `countdown` that counts down from 10 on the console and
  * then logs "Blast Off!"
  */
+ function countdown() {
+ 	for (var i = 10; i >= 0; i--) {
+ 		console.log(i);
+ 		if (i <= 0) {
```

Comment:
This totally works as a solution. Just wanted to point out that since this loop terminates when `i` is 0, we know that `i` can never be less than 0. Therefore, we only _really_ need to check for equality:

```
if (i === 0) {
   console.log('Blast Off!!!');
}
```